docs/source/installation/migration/1-migrating-project.rst

```diff
@@ -125,30 +124,40 @@ will be unable to submit forms or sync with the server.
   * ``./manage.py dump_domain_data <domain_name>`` 
   * ``./manage.py run_blob_export --all <domain_name>``
 
-* Transfer these two files to the new environment.
+  .. note::
+     It is important to have the commit hash that ``dump_domain_data`` and ``run_blob_export`` were run from. If
+     Dimagi does not provide you with this commit hash, please followup to ensure you are able to reference this
+     hash in future steps.
+
+* Transfer these files to the new environment.
 
 .. note::
-  If you are not able to use your own domain for a test run and would like dump data for a test domain for practising or testing, please contact us via https://forum.dimagi.com/c/developers/ with subject "Request for test domain dump data for migration testing" and mention this page. A dimagi developer will provide you above data for any test/QA domains (casesearch, ccqa, dataregistry, qateam, ben-test, qa-erm-v1-downstream1) from https://staging.commcarehq.org.
+  If you are not able to use your own domain for a test run and would like dump data for a test domain for
+  practising or testing, please contact us via https://forum.dimagi.com/c/developers/ with subject
+  "Request for test domain dump data for migration testing" and mention this page. A dimagi developer will
+  provide you above data for any test/QA domains (casesearch, ccqa, dataregistry, qateam, ben-test, qa-erm-v1-downstream1)
+  from https://staging.commcarehq.org.
 
 
 3. Prepare the new environment to be populated
 ----------------------------------------------
 
+* Ensure you are running the following steps from a release created using the CommCare version/commit hash that you
+  should have been provided in Step 1. This ensures the database will be migrated to the same state it was in when
+  the data was dumped.
```

Comment:
thanks for adding this @gherceg 

There is one challenge though. The [documented for deploy](https://commcare-cloud.readthedocs.io/en/latest/installation/2-manual-install.html#deploy-commcare-hq-code) does not say how it is to be done. So, we would need to update there as well.

Reply:
Thanks Manish I'll make a new PR as there are some other changes I'd like to make too. Feel free to add any additional comments you may still have and I'll address them there! I admittedly rushed this out just to get something closer to up to date instructions out there.